pkg/controller/workload/cache/endpoint_cache.go

```diff
@@ -30,7 +30,11 @@ type Endpoint struct {
 type EndpointCache interface {
 	List(uint32) map[uint32]Endpoint // Endpoint slice by ServiceId
 	AddEndpointToService(ep Endpoint, serviceID uint32)
+	// DeleteEndpoint delete a endpoint regardless of the priority
 	DeleteEndpoint(workloadID, serviceID uint32)
```

Comment:
`DeleteEndpoint` maybe not used.

Reply:
Good catch, this is only used in test now, will change that